package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@vscode/jupyter-ipywidgets7",
-    "version": "1.0.3",
+    "version": "2.0.0",
```

Comment:
Introduced some API changes (changes to how the script is loaded an the API to communicate with this)
Hence the major bump